demos/text-field.html

```diff
@@ -275,60 +271,71 @@ <h2>Text Field box</h2>
 
       <section class="example">
         <h2>Text Field - Leading/Trailing icons</h2>
-        <div class="demo-note">
```

Comment:
Are we sure about removing this advice? I think the intent was to tell people how to make text fields consistent size both with and without icons in them, though that's not really visible from this demo example alone since we only use the class on the icon examples. It might make more sense to move this to a note within the icon readme?

Reply:
Yep, Alex confirmed it's fine since the input elements have `width:100%`.
Making text fields consistent size both with and without icons is done the same way (by setting the width of the top-level `mdc-text-field` element).